spec/index.bs

```diff
@@ -7285,7 +7308,13 @@ dictionary GPUQuerySetDescriptor : GPUObjectDescriptorBase {
 
             **Returns:** {{undefined}}
 
-            Issue: Describe {{GPUQuerySet/destroy()}} algorithm steps.
+            If any of the following conditions are unsatisfied, generate a validation error and stop.
+            <div class=validusage>
+                - |this| is [$valid to use with$] |this|.{{GPUObjectBase/[[device]]}}
+            </div>
```

Comment:
I don't think this validation rule is needed. All it does is check that that the query set is not invalid. For buffers, I don't think we have this requirement, so it is allowed to destroy invalid buffers.

```suggestion
```